RHEL6_7/storage/multipath/check

```diff
@@ -23,11 +23,11 @@ if [ -f "$MULTIPATH_CONF" ]; then
 		ERR=1
 	fi
 	if grep -q "devices[[:space:]]*{" $MULTIPATH_CONF; then
-		log_high_risk "Multipath matches user device configs with built-in device configs differently in Red Hat Enterprise Linux 7."
+		log_warning "Multipath matches user device configs with built-in device configs differently in Red Hat Enterprise Linux 7. This will be taken care of by the postupgrade script."
 		FIX=1
 	fi
 fi
-log_medium_risk "The number of default configuation settings have changed in Red Hat Enterprise Linux 7. Verify that your devices are still optimally configured."
+log_warning "The number of default configuation settings have changed in Red Hat Enterprise Linux 7. Verify that your devices are still optimally configured."
```

Comment:
Doesn't make sense to use `log_warning` when message obviously say that user should do something later. This is obviously `log_medium_risk`.

Reply:
Same as the above